### DISCUSSION TITLE
Bank Tags: Is it possible to allow new highscore/league pixel art for bosses to be used for icons.

### DISCUSSION THREAD

**Discussion by Luke-Chatburn** [runelite/runelite#18605] [Ideas]
Would be great if we can use the pixel art for tab icons

**Comment by Nightfirecat** [comment]
While it wouldn't be hard to access those sprites for tab icons, the more difficult part of this would be updating the UI/UX to support it, since those icons (we refer to them as sprites) do not have names associated with them, unlike items, so making them searchable or easily scannable would be the difficult part of presenting them for selection.

**Reply by Luke-Chatburn** [reply]
Cheers, I understand the complexity, would be cool if anyone ever fancies giving it a bash or this gets more demand.
